sql/api/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseLexer.g4

```diff
@@ -576,6 +576,10 @@ BACKQUOTED_IDENTIFIER
     : '`' ( ~'`' | '``' )* '`'
     ;
 
+LABEL
```

Comment:
please check "Side changes" section of the PR description to see why we had to introduce this change